actors/builtin/reward/reward_policy.go

```diff
@@ -0,0 +1,3 @@
+package reward
+
+var ExpectedLeadersPerRound = int64(5)
```

Comment:
This might belong in `builtin/network.go` (but nbd if here for now)

Reply:
For consistency, `ExpectedLeadersPerEpoch`?